[Docs]: Linked to Organisation Contribution Guide (#161)
Transport-for-the-North/caf.toolkit: README.md

[NorMITs Demand](https://github.com/Transport-for-the-North/NorMITs-Demand)
more widely available and easily accessible.

## Common Analytical Framework

This package is sits within the [Common Analytical Framework (CAF)](https://transport-for-the-north.github.io/caf_homepage/intro.html),
which is a collaboration between transport bodies in the UK to develop and maintain commonly used
transport analytics and appraisal tools.

## Contributing
CAF.Toolkit happily accepts contributions.

The best way to contribute to this project is to go to the [issues tab](https://github.com/Transport-for-the-North/caf.toolkit/issues)

> and report bugs or submit a feature request. This helps CAF.toolkit become more

Reviewer: ```suggestion
to report bugs or submit feature requests. This helps CAF.toolkit become more
```
